### DISCUSSION TITLE
Why the server weighs as much

### DISCUSSION THREAD

**Discussion by gitFoxCode** [kus/cs2-modded-server#30] [Q&A] [answered]
Why server weighs as much as as 60 GB (need to have 60BG of free space)? Game itself is around 30GB

**Comment by kus** [comment]
You can get away with 40GB. I just say 60GB to give room for Competitive match demos (gotv recordings) and custom maps as the server downloads them.

The actual repo itself is 220MB~ and the bulk of that is [CounterStrikeSharp ](https://github.com/roflmuffin/CounterStrikeSharp/releases) as I include Linux and Windows.

**Reply by gitFoxCode** [reply]
Hey, thanks for reply, Is there a minimum requirement? Can I disable the demo and download the map from the workshop only when needed? (As far as I know, this is how free hosting for CS2 websites works.) I would like to create a private server similar to faceit for friends 5vs5, but I don't want to allocate even 10GB for it

**Reply by kus** [reply, marked as answer]
I would say don't go lower than 40GB. A fresh install of CS2 on Ubuntu with my mod also installed is 34GB. You also need to allow space for the OS itself.

If you set `tv_enable 0` in your custom files for competitive it should stop it from recording. Please thouroughly read the instructions before asking for help.

**Reply by creazy231** [reply]
in addition: keep in mind that workshop maps also take space

**Reply by kus** [reply]
Yep, custom maps are quite big these days. On my Ubuntu server (which is only for the CS2 server) I have 60Gb HDD and 5Gb free, hence the recommended 60Gb minimum. But if you are only playing Valve maps you could get away with 40Gb.
